apps/mark-scan/frontend/src/pages/ask_poll_worker_page.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { Screen, H1, Main, Button, Text } from '@votingworks/ui';
+
+import { InsertedSmartCardAuth } from '@votingworks/types';
+import { isCardlessVoterAuth } from '@votingworks/utils';
+
+import { ButtonFooter } from '../components/button_footer';
+
+interface Props {
+  authStatus:
+    | InsertedSmartCardAuth.CardlessVoterLoggedIn
+    | InsertedSmartCardAuth.PollWorkerLoggedIn;
+  cardlessVoterContent: {
+    body: JSX.Element;
+  };
+  pollWorkerContent: {
+    headerText: string;
+    body: JSX.Element;
+    buttonText: string;
+    onButtonPress: () => Promise<void>;
+  };
+}
+
+// Component with 2 stages
+// 1. Voter stage: tells the voter to ask a poll worker for help. Does not allow the voter to advance the state.
+// 2. Poll worker stage: after auth, gives instructions and a button to advance the state.
+export function AskPollWorkerPage(props: Props): JSX.Element {
+  const { authStatus, cardlessVoterContent, pollWorkerContent } = props;
+
+  let mainContents = (
+    <React.Fragment>
+      <H1>{pollWorkerContent.headerText}</H1>
+      {pollWorkerContent.body}
+    </React.Fragment>
+  );
+  let button = (
+    <Button onPress={() => pollWorkerContent.onButtonPress()}>
+      {pollWorkerContent.buttonText}
+    </Button>
+  );
+
+  if (isCardlessVoterAuth(authStatus)) {
+    mainContents = (
+      <React.Fragment>
+        <H1>
+          <span aria-label="Ask a Poll Worker for Help">
+            Ask a Poll Worker for Help
+          </span>
+        </H1>
+        {cardlessVoterContent.body}
+      </React.Fragment>
+    );
+    button = (
+      // onPress is a required prop but we want the button to no op
+      // eslint-disable-next-line @typescript-eslint/no-empty-function
+      <Button disabled onPress={() => {}}>
+        Insert a Poll Worker Card to Continue
+      </Button>
```

Comment:
I wonder if this should just be a plain text message instead of a disabled button, since it's not really asking for the user to press the button, but rather do something off screen

Reply:
Good point. I added this as a button because I thought it would make it clearer that a poll worker card is required to continue. While I still think it does that more effectively than a text message (that people might skim over), it's somewhat of a dark pattern because the button will never actually work. It also might (?) be less accessible to screen readers. Removed in [b2aba7c](https://github.com/votingworks/vxsuite/pull/3962/commits/b2aba7c9a11c378009cdda4f0298e03c970fce88)